src/network/NetworkController.ts

```diff
@@ -18,7 +19,9 @@ export type NetworkType =
   | 'ropsten'
   | 'rpc'
   | 'optimism'
-  | 'optimismTest';
+  | 'optimism-mainnet'
```

Comment:
Types were added for `eth-json-rpc-infura`. One of those types is `SupportedInfuraNetwork`, which is a whole list of networks that Infura supports (i.e. that can be plugged into `https://${networkName}.infura.io`). `optimism-mainnet` and `optimism-kovan` are two of those networks. `optimism` and `optimismTest` are NOT valid networks, but are left here for backward compatibility.